sql_metadb/derived_tables/instance_alternative_titles.sql

```diff
@@ -0,0 +1,26 @@
+DROP TABLE IF EXISTS instance_alternative_titles;
+
+CREATE TABLE instance_alternative_titles AS 
+SELECT 
+	i.id AS instance_id,
+	jsonb_extract_path_text(i."jsonb", 'hrid') AS instance_hrid,
+	jsonb_extract_path_text(titles.jsonb, 'alternativeTitle') AS alternative_title,
+    jsonb_extract_path_text(titles.jsonb, 'alternativeTitleTypeId')::uuid AS alternative_title_type_id,
+    titles.ORDINALITY AS alternative_title_ordinality,
```

Comment:
Here `ordinality` is the name of a column (I know it is confusing) and so it should be all in lowercase.

Reply:
DBeaver automatically puts this in uppercase. I can change it to lowercase.